frontend/src/app/components/patient-data/patient-data.component.html

```diff
@@ -82,7 +82,8 @@
         <app-patient-data-item>
           <mat-label>Validováno v ISIN:</mat-label>
           <mat-label>
-            <app-chip [value]="!!patientData.isinId"></app-chip>
+            <app-chip [value]="!!patientData.isinId && !!patientData.isinReady"
```

Comment:
Tady podle me ma byt OR. Pokud nemame id nebo neni ready, chceme ukazovat ze Validovano v ISIN: NE

Reply:
`!!` je negace negace. Když máme isin id && je to ready, ukazujeme ANO

Reply:
A to jako Angular vyzaduje, kdyz chci pracovat s nejakou boolean, abych udelal negaci negace? `patientData.isinReady` nefunguje, jo?

Reply:
Ono to může být i null a tímhle se to převede na false

Reply:
aha, ok. To jsou triky tedy...